sensor_msgs/PointCloud2 single_cluster
---
bool match
